compio-driver/src/fusion/mod.rs

```diff
@@ -0,0 +1,169 @@
+#[path = "../poll/mod.rs"]
+mod poll;
+
+#[path = "../iour/mod.rs"]
+mod iour;
```

Comment:
Why `path` instead of using normal imports?

Reply:
It's in parent dir.

Reply:
`pub(crate)` allows you to access that if necessary

Reply:
No it's parent **directory**, and have not been mod'd.